st2actions/st2actions/workflows/workflows.py

```diff
@@ -54,6 +54,10 @@ def handle_workflow_execution_with_instrumentation(wf_ex_db):
                 return self.handle_workflow_execution(wf_ex_db=wf_ex_db)
 
         def handle_action_execution_with_instrumentation(ac_ex_db):
+            # Ignore non orquesta workflow executions
+            if 'orquesta' not in ac_ex_db.context:
```

Comment:
To clarify, same thing already happens here - https://github.com/StackStorm/st2/pull/4323/files#diff-b528f98088d2afaa8a99f531b71fdd82R164.

I intentionally used a wrapper / decorator function for instrumentation purposes. Adding instrumentation directly to the handler method would result in more complexity and code which is harder to read and manage (whole chunk of code would be indented by one additional level).